compilers/esm.js

```diff
@@ -101,7 +101,7 @@ exports.compile = function(load, opts, loader) {
       lowResolutionSourceMap: opts.lowResSourceMaps
     });
 
-    var tree = load.metadata.parseTree || compiler.parse(load.source, load.path);
+    var tree = load.metadata.parseTree || compiler.parse(source, load.path);
```

Comment:
Haha. Better change it! It's all good, as I was able to fix this for myself locally to continue development until the patch gets released upstream :sweat_smile: